src/Business Foundation/App/NoSeries/src/Legacy/NoSeriesLineObsolete.TableExt.al

```diff
@@ -0,0 +1,61 @@
+// ------------------------------------------------------------------------------------------------
```

Comment:
This tableextension looks very weird. I understand that they're obsoleted elements from the `No. Series` table, but now wouldn't we be stuck with a tableextension that does nothing once functions go away?

Why couldn't this be a codeunit which can be removed entirely?